pkg/domain/domain.go

```diff
@@ -158,8 +158,10 @@ type Domain struct {
 	m               syncutil.Mutex
 	SchemaValidator SchemaValidator
 	schemaLease     time.Duration
-	sysSessionPool  util.SessionPool
-	exit            chan struct{}
+	// Note: If you no longer need the session, you must call Destroy to release it.
+	// Otherwise, the session will be leaked. Because there is a strong reference from the session to the domain.
```

Comment:
> Because there is a strong reference from the session to the domain.

Should be "from the domain to the session"?

Reply:
Oops!